sensorlib/src/sensor/SemanticLidarSensor.py

```diff
@@ -519,3 +522,28 @@ def update_object_frame_and_timestamps_from_hitpoint(self, obj, timestamp):
                        timestamp=timestamp,
                        position=new_position
                        )
+
+    def update_velocity(self, detected_objects):
```

Comment:
This function will be impact more than just sumo vehicles right. This will replace all velocities.  @MishkaMN is this what we want to do. Replace all velocities with a velocity estimate.

Reply:
not really, inside there is a condition it will only change the ones that have 0 velocity

Reply:
I was thinking we should add extra condition where this velocity change should only apply to objects whose _position changed from previous frame despite their velocity being 0_